index.html

```diff
@@ -0,0 +1,208 @@
+    <div id="portfolio-body" class="container column">
+        <p>I've fallen in love with React and
+            Javascript in general in the last few years. I first got my feet wet
+            writing code in Excel VBA (custom calculators for various subjects)
+            and that got me interested in doing something similar in the browser.
+            I developed my first Chrome extension soon after giving me access to a
+            robust calculator wherever I find myself online, allowing me to
+            streamline coursework.</p><p>Since then, I've delved into various projects to help me learn more
+            about Javascript, including a Pacman clone, a private online
+            whiteboard, and an app that I use to keep me and my workers organized
+            when working on coursework.
+        </p>
+        <div class="inner"><div class="project_head"><h1>All You Can Calc</h1></div><div class="project_item"><div class="project_description"><div role="tablist" aria-orientation="horizontal"><button id="headlessui-tabs-tab-:r0:" role="tab" type="button" aria-selected="true" tabindex="0" data-headlessui-state="selected" aria-controls="headlessui-tabs-panel-:r3:">Description</button><button id="headlessui-tabs-tab-:r1:" role="tab" type="button" aria-selected="false" tabindex="-1" data-headlessui-state="" aria-controls="headlessui-tabs-panel-:r4:">Example 1</button><button id="headlessui-tabs-tab-:r2:" role="tab" type="button" aria-selected="false" tabindex="-1" data-headlessui-state="" aria-controls="headlessui-tabs-panel-:r5:">Example 2</button></div><div><div id="headlessui-tabs-panel-:r3:" role="tabpanel" tabindex="0" data-headlessui-state="selected" aria-labelledby="headlessui-tabs-tab-:r0:"><p>This is a Chrome Extension I created several years ago that I still use on a near-daily basis. I do a lot of Statistics, Finance, and Trigonometry homework, so having a calculator in the browser speeds up all over my work.</p><p>Each solver is given multiple equations with flags that identify which equation(s) to use based on the variables and their values. The solvers implement Newton's method to solve equations if only one variable is missing.</p></div><span id="headlessui-tabs-panel-:r4:" role="tabpanel" tabindex="-1" style="position: fixed; top: 1px; left: 1px; width: 1px; height: 0px; padding: 0px; margin: -1px; overflow: hidden; clip: rect(0px, 0px, 0px, 0px); white-space: nowrap; border-width: 0px;" aria-labelledby="headlessui-tabs-tab-:r1:"></span><span id="headlessui-tabs-panel-:r5:" role="tabpanel" tabindex="-1" style="position: fixed; top: 1px; left: 1px; width: 1px; height: 0px; padding: 0px; margin: -1px; overflow: hidden; clip: rect(0px, 0px, 0px, 0px); white-space: nowrap; border-width: 0px;" aria-labelledby="headlessui-tabs-tab-:r2:"></span></div></div><div class="project_code"><iframe class="project_iframe" src="https://codepen.io/RickCedWhat/embed/ZEVpwKz" width="1200" height="600"></iframe></div></div></div>
```

Comment:
these blocks of code are pretty hard to read. I know that a good portion of it came from embedding a CodePen - consider making separate lines for the code not coming from the embedding process so that it's easier to discern.